Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -193,22 +196,24 @@ class HomeFragmentViewModel @AssistedInject constructor(
             when (statePCR) {
                 SubmissionStatePCR.NoTest -> {
                     if (stateRAT == SubmissionStateRAT.NoTest) {
-                        add(testPCR.toTestCardItem())
+                        add(testPCR.toTestCardItem(testPCR?.identifier ?: ""))
```

Comment:
Dont repeat yourself
Create a variable for _identifier_ like: val _identifier_ = testPCR?.identifier ?: ""

Reply:
Alright, will do that.